source/state_representation/src/space/Jacobian.cpp

```diff
@@ -300,6 +302,7 @@ Jacobian operator*(const CartesianPose& pose, const Jacobian& jacobian) {
 }
 
 double& Jacobian::operator()(unsigned int row, unsigned int col) {
+  // FIXME what do we do here?
```

Comment:
Here I think we have to be more careful. In my opinion, we should the Jacobian to filled and reset the timestamp. What do you think?

Reply:
It's non-const so the timestamp should be reset. Let's be completely consistent in that regard. There is a const accessor as an alternative where the timestamp wouldn't be reset.

With regards to set filled or not, keep the empty assertion in the line below and _don't_ set filled. This prevents a user from setting scalar data by reference on an otherwise empty Jacobian.

i.e. the following would (rightly) throw an exception: 

```c++
Jacobian jac();
jac(1, 1) = 1.0;
```

Reply:
Regarding set_empty, I do agree 100%